src/Terminal/HelpText.ts

```diff
@@ -14,6 +14,7 @@ export const TerminalHelpText: string[] = [
   "    connect [hostname]               Connects to a remote server",
   "    cp [src] [dest]                  Copy a file",
   "    download [script/text file]      Downloads scripts or text files to your computer",
+  "    echo [string]                    Print the specified string to the terminal. Typically used for piping.",
```

Comment:
```suggestion
  "    echo [string]                    Print the specified string to the terminal.",
```
The second sentence is unneeded, especially in a short description.